src/panels/lovelace/components/hui-image.ts

```diff
@@ -66,11 +70,37 @@ export class HuiImage extends LitElement {
     if (this.cameraImage && this.cameraView !== "live") {
       this._startUpdateCameraInterval();
     }
+    this._intersectionObserver = new IntersectionObserver(
```

Comment:
We should only init the `IntersectionObserver` when there is a `cameraImage && cameraView !== "live"`

Reply:
done